### DISCUSSION TITLE
Deploy  and verify to zksync

### DISCUSSION THREAD

**Discussion by Dean-address** [Cyfrin/foundry-full-course-cu#3090] [Q&A]
Hi everyone, i try to deploy and verify to zsync but I keep getting this error but If I just deploy without verifying I don't get error. Please I would really appreciate any help yall can render
![Screenshot (15)](https://github.com/user-attachments/assets/b7ebeac1-dc60-4358-9a15-e63bd49458dd)

**Comment by EngrPips** [comment]
Hello, clear your browser cache, refresh `Remix`, and then retry. Let's see i the issue persists

**Reply by Dean-address** [reply]
error is still the same

**Reply by EngrPips** [reply]
Share your code let me try on my end. Please format your code nicely.

**Reply by Dean-address** [reply]
### Thank you so much. this is the code below
`// I'm a comment!
// SPDX-License-Identifier: MIT

pragma solidity 0.8.24;

contract SimpleStorage {
    uint256 myFavoriteNumber;

    struct Person {
        uint256 favoriteNumber;
        string name;
    }
    // uint256[] public anArray;
    Person[] public listOfPeople;

    mapping(string => uint256) public nameToFavoriteNumber;

    function store(uint256 _favoriteNumber) public {
        myFavoriteNumber = _favoriteNumber;
    }

    function retrieve() public view returns (uint256) {
        return myFavoriteNumber;
    }

    function addPerson(string memory _name, uint256 _favoriteNumber) public {
        listOfPeople.push(Person(_favoriteNumber, _name));
        nameToFavoriteNumber[_name] = _favoriteNumber;
    }
}`

**Reply by EngrPips** [reply]
I was able to deploy but for one reason or the other, which I don't know, `Remix` did not allow me to check the `verify` box. I will urge you not to let this block you as it is not compulsory that you verify the contract. The deployment is sufficient to interact with the contract and learn what is taught.

**Reply by Dean-address** [reply]
Exactly but when I try to check my hash address the zfsync is saying I should verify to check my contract and I get error too

**Reply by Dean-address** [reply]
I guess I will just continue like and I hope the error Dont occur again. Thank you @EngrPips for your time I really do appreciate

**Reply by EngrPips** [reply]
You don't need to see your contract codebase on zksync etherscan since you know what the codebase is already. Just interact with the codebase on `Remix` and you are good.

**Reply by Dean-address** [reply]
Okay
